src/components/widgets/RaceTrackerMap/RaceTrackerMap.tsx

```diff
@@ -126,7 +130,10 @@ class RaceTrackerMap extends React.Component<IRaceTrackerMapProps, any> {
         mapboxApiAccessToken={TOKEN}
         mapStyle={this.state.mapStyle}
         {...this.state.viewport}
-        onViewportChange={this.__handleViewportChange}>
+        onViewportChange={this.__handleViewportChange}
+        onClick={this._onClick}>
+        
+         {this._renderPopup()}
```

Comment:
convention for `private` functions is 2 _ in front of the name:
`this._renderPopup()` -> `this.__renderPopup()`